CHANGELOG.md

```diff
@@ -40,15 +42,6 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   navigational element by screen readers,
 - Fix add plugin to team and organization on fragment_course_content template
   when they are empty,
-- Remove deprecated attribute "type" from some "<script>" tags in templates,
-- Add missing attribute "alt" on some "<img>" tags in templates,
-- Fix invalid syntax in "srcset" attribute value on "<img>" tags,
-- Add url escaping on variables inside links on social network badges,
-- Fix invalid syntax in "sizes" attribute value on "<img>" tags,
-- Fix invalid attributes on "iframe" tag from djangocms_video plugin,
-- Fix invalid "button" tag inside a "a" tag inside course glimpse,
-- Fix invalid "role" value on pagination list with a "nav" tag around bullet
-  list instead,
```

Comment:
Bad granularity? Should it be replaced by a more generic message, _e.g._ "Fixed DOM validation errors"?

Reply:
Arf. Ok then...

Reply:
@jmaupetit we agree on this I will modify it!